src/plugins/Workers/TransactionSyncStreamWorker/methods/getLastSyncedBlockHash.spec.js

```diff
@@ -0,0 +1,32 @@
+const {expect} = require('chai');
+const getLastSyncedBlockHash = require('./getLastSyncedBlockHash');
+
+const mockStore = {
```

Comment:
Can we create a function that returns store mock, so we don't have to change this place everywhere if we add/remove something from the store?

Reply:
improving mock in wallet-lib seems like a great idea for our backlog :D